### ISSUE TITLE
Add a map

### ISSUE THREAD

**Issue by Discostu36** [Discostu36/OSM-Photos#2]
This is the first step for creating the app. Deciding for a solution for displaying the background map and, later, the data layer (#3). 

A link dump for this task:
- https://github.com/osmlab/editor-layer-index
- https://stackoverflow.com/questions/15946316/use-leaflet-in-an-android-application-to-show-online-map
- https://forum.openstreetmap.org/viewtopic.php?id=61005
- https://wiki.openstreetmap.org/wiki/Mapsforge

**Comment by Discostu36** [comment]
As it is free for up to 25.000 monthly users, Mapbox might be an alternative.

https://docs.mapbox.com/android/maps/guides/

**Comment by Discostu36** [comment]
As Mapbox is not FOSS, the app would be rejected by FDroid.

[MapLibre](https://maplibre.org/projects/maplibre-native/) might be a suitable alternative ([installation](https://github.com/maplibre/maplibre-gl-native#android), [video tutorial](https://www.youtube.com/watch?v=VzQEbGyGe4E)).

[OSMDroid](https://github.com/osmdroid/osmdroid) would be another alternative.

Choice of a tile server is the next question. For initial development, I guess it will be ok to use osm.org tiles.

**Comment by JLZIMMERMANN** [comment]
The map could show where thare have already been pictures taken, and where there could be pictures missing.

**Comment by Discostu36** [comment]
Yes, this is tackled in #3. This issue is about the framework and tile server to use for the background map. 

I think that OSMDroid is the best choice as it also has features for adding data layers. But I haven't been able to get it running yet.

This is my first Android project, so the first steps might take a while until I have gathered sufficient knowledge to get things started.

If someone is impatient and wants to make a pull request for OSMDroid integration, feel free to do so.
